x-pack/plugins/rollup/public/crud_app/sections/job_list/job_table/job_table.js

```diff
@@ -253,7 +253,6 @@ export class JobTableUi extends Component {
       if (name === 'ID') {
         content = (
           <EuiLink
-            data-test-subj="rollupTableJobLink"
```

Comment:
Is it redundant? Maybe we can use it in other UI tests

Reply:
kept it specific to that test, as I will reuse for other rollup job tests again.